packages/suite/src/constants/wallet/sendForm.ts

```diff
@@ -41,7 +41,8 @@ export const DEFAULT_OPRETURN = {
     dataHex: '',
 } as const;
 
-export const DEFAULT_OPTIONS = ['broadcast', 'bitcoinRBF'] as const;
+// export const DEFAULT_OPTIONS = ['broadcast', 'bitcoinRBF'] as const;
+export const DEFAULT_OPTIONS = ['broadcast'] as const;
```

Comment:
The goal of the feature flag is to flip a single switch to enable/disable a feature. This defeats the purpose of it as it requires code changes.

Reply:
true, updated